js/products.js

```diff
@@ -0,0 +1,76 @@
+// find and show selected product
+
+let foundProduct = {};
+let totalWeight = null;
+let sumKcalProduct = null;
+let totalKkal = null;
+
+productSelectElement.addEventListener("input", (event) => {
+  foundProduct = products.find(
+    (product) => product.name === event.target.value
+  );
+  if (foundProduct) {
+    productElement.className = "active";
+    nameProductElement.textContent = foundProduct.name;
+    imageProductElement.src = foundProduct.imageUrl;
+    kcalProductElement.textContent = foundProduct.kcal;
+    proteinsProductElement.textContent = foundProduct.proteins;
+    fatsProductElement.textContent = foundProduct.fats;
+    carbohydratesProductElement.textContent = foundProduct.carbohydrates;
+    updateThis(productSelectElement, null);
+  }
```

Comment:
All this element setting can be moved into the function and made as buildPopupProductElement with injected html inside js file.